### PR TITLE
Support emoji

### PR DESCRIPTION
**Pull request by tkatox**
・support emoji #13
・python to python3

### CODE REVIEW

File: papery/data/sample/themes/default/assets/css/style.css [hunk 1]
```diff
@@ -79,3 +79,6 @@ table tr td {
     border-bottom: 1px solid #e1e1e1;
 }
 
+.emojione, .twemoji, .gemoji {
+  height: 1em;
```
Comment (na4da) [inline papery/data/sample/themes/default/assets/css/style.css:83]:
```suggestion
  display: inline-flex;
  height: 1.25em;
  vertical-align: text-top;
```

By applying the emoji style that's actually used on [PyMdown Extensions Documentation](https://facelessuser.github.io/pymdown-extensions/extensions/emoji/), I think it looks better.

**Review by na4da — CHANGES_REQUESTED** [review]
Thank you for working on this! I confirmed that emoji is displayed on sample page by `papery run`.

**Review by na4da — APPROVED** [review]
LGTM 👍

File: papery/page.py [hunk 2]
```diff
@@ -49,11 +50,20 @@ def text(self, lang=None, link=None):
...
 
+        html = markdown.markdown(text,
+                                 extensions=["tables", "pymdownx.emoji"],
+                                 extension_configs = {
+                                     "pymdownx.emoji": {
+                                         "emoji_index": pymdownx.emoji.gemoji,
+                                         "emoji_generator": pymdownx.emoji.to_png,
+                                         "alt": "short",
+                                         "options": {
+                                             # TODO make "image_path" configurable by papery's configuration file for security reasons
```
Comment (federkasten) [inline papery/page.py:61]: I don't want to load emoji files from a CDN for security reasons especially for internal sites.

**Review by federkasten — APPROVED** [review]

**Pull request merged**